BusinessLogic/Services/UserService.cs

```diff
@@ -0,0 +1,37 @@
+﻿using ApiModels.Models;
+using AutoMapper;
+using AutoMapper.Internal;
+using DataAccess.Repository;
+using DataAccess.Repository.Interfaces;
+using Domain.EF_Models;
+using Microsoft.AspNetCore.Identity;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BusinessLogic.Services
+{
+    public class UserService
```

Comment:
this should be an implementation of IUserService interface, so add it with appropriate methods, register it in the dependency injection and inject to the controller IUserService